docs/vdrtools-cli/vdrtools-cli-identity-tutorial.md

```diff
@@ -0,0 +1,369 @@
+
+  
+
+In order to create a DID using the instructions outlined in this tutorial, you must be using Ubuntu 20.04 terminal. You'll find all the information required to setup your Ubuntu 20.04 terminal at the end of this tutorial.
+
+  
+
+Please ensure you are running the correct version of testnet. You can check which is the current version of testnet [here](https://rpc.testnet.cheqd.network/abci_info?).
+
+  
+
+### Ensure you have the correct OS environment
+
+  
+
+If you don't currently have Ubuntu 20.04 installed on your machine you can use VirtualBox or [Docker](#docker-setup)
+
+  
+
+### Install VDR Tools on Ubuntu 20.04
+
+  
+
+After the preparation of the Ubuntu 20.04 terminal please make sure you have installed VDR tools.
+
+  
+
+In general this process is described on the main page of VDR tools [repository](https://gitlab.com/evernym/verity/vdr-tools#installing) but to be short let's make the next steps inside Ubuntu 20.04:
+
+  
+
+#### 1\. Install additional packages:
+
+  
+
+```
+$ apt install curl libsodium23 libzmq5 libncursesw5-dev -y
+```
+
+#### 2\. Download and install `libvdrtools`:
+
+  
+
+```
+$ curl https://gitlab.com/evernym/verity/vdr-tools/-/package_files/27311917/download --output libvdrtools_0.8.4-focal_amd64.deb && dpkg -i libvdrtools_0.8.4-focal_amd64.deb
+```
+
+#### 3\. Download and install `vdrtools-cli`:
+
+  
+
+```
+$ curl https://gitlab.com/evernym/verity/vdr-tools/-/package_files/27311922/download --output vdrtools-cli_0.8.4-focal_amd64.deb && dpkg -i vdrtools-cli_0.8.4-focal_amd64.deb
+```
+
+  
+
+Once this is installed we're now ready to write a DID to the cheqd network.
+
+  
+
+Writing a DID to the cheqd network
+----------------------------------
+
+  
+
+All the command described below are supposed to be used from VDR Tools CLI.
+
+For running VDR Tools terminal just type:
+
+  
+
+```
+$ vdrtools-cli
+```
+
+  
+
+The next steps are about how to send a DID.
+
+  
+
+#### 1\. Create a wallet
+
+  
+
+The command:
+
+  
+
+```
+wallet create cheqd-wallet key 
+```
+
+  
+
+Example:
+
+  
+
+```
+indy> wallet create cheqd-wallet key
+Enter value for key:
+
+Wallet "cheqd-wallet" has been created
+```
+
+  
+
+where `Enter value for key` is a suggestion to create a secure password.
+
+  
+
+#### 2\. Open your wallet: 
+
+  
+
+Next use the following command to open the wallet created in the previous step
+
+  
+
+```
+wallet open cheqd-wallet key
+```
+
+  
+
+Within this command you will need to enter the password you have just created.
+
+  
+
+Example:
+
+  
+
+```
+indy> wallet open cheqd-wallet key
+Enter value for key:
+
+Wallet "cheqd-wallet" has been opened
+```
+
+  
+
+#### 3\. Create a pool connection: 
+
+  
+
+Now we're ready to create a pool connection. This is a configuration to allow to connection to the of your wallet to the pool.
+
+  
+
+Use this command:
+
+  
+
+```
+cheqd-pool add alias=cheqd-pool-testnet rpc_address=https://rpc.testnet.cheqd.network/ chain_id=cheqd-testnet-4
```

Comment:
Do you need to add the port here or is it implicit?

Reply:
it's implicit